packages/sync/src/utils/buildProjectLinkedLookupQueryDetails.ts

```diff
@@ -0,0 +1,9 @@
+import { DatabaseModel } from '@tupaia/database';
+
+import { buildSyncLookupSurveyProjectIdSelect } from './buildSyncLookupUtils';
+
+export async function buildProjectLinkedLookupQueryDetails(model: DatabaseModel) {
```

Comment:
This one feels a little small to necessitate a separate file?

Reply:
I was thinking that beside survey, there are a few tables that directly linked with projects, so would be convenient if we have a util like this that can be used in all of those models. However I just double checked and beside survey there's only 2 more which seems insignificant. Happy to remove it!